service-sdk/12.0.0/migrate-to-open-api/messaging-service/pom.xml

```diff
@@ -20,19 +21,11 @@
     <name>Backbase :: Digital Banking Services :: example-service</name>
 
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>11</java.version>
+        <boat-maven-plugin.version>0.14.0</boat-maven-plugin.version>
     </properties>
 
     <dependencies>
-
-        <!-- Added spec dependency -->
-        <!-- tag::spec-dependency[] -->
```

Comment:
Removing tag causes asciidoc build to break, can't merge until I've taken this out of asciidoc